Traceback (most recent call last):
  File "/Users/kensuke/venv/lib/python3.9/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/Users/kensuke/venv/lib/python3.9/site-packages/nbclient/client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/Users/kensuke/venv/lib/python3.9/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/Users/kensuke/venv/lib/python3.9/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/Library/Developer/CommandLineTools/Library/Frameworks/Python3.framework/Versions/3.9/lib/python3.9/asyncio/base_events.py", line 642, in run_until_complete
    return future.result()
  File "/Users/kensuke/venv/lib/python3.9/site-packages/nbclient/client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "/Users/kensuke/venv/lib/python3.9/site-packages/nbclient/client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/Users/kensuke/venv/lib/python3.9/site-packages/nbclient/client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
# This will be an error. You can't add a number and a string. 
# You need to convert the string to a number or vice versa.
print(3+'4')
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mTypeError[0m                                 Traceback (most recent call last)
Cell [0;32mIn [5], line 3[0m
[1;32m      1[0m [38;5;66;03m# This will be an error. You can't add a number and a string. [39;00m
[1;32m      2[0m [38;5;66;03m# You need to convert the string to a number or vice versa.[39;00m
[0;32m----> 3[0m [38;5;28mprint[39m([38;5;241;43m3[39;49m[38;5;241;43m+[39;49m[38;5;124;43m'[39;49m[38;5;124;43m4[39;49m[38;5;124;43m'[39;49m)

[0;31mTypeError[0m: unsupported operand type(s) for +: 'int' and 'str'
TypeError: unsupported operand type(s) for +: 'int' and 'str'

